CODEOWNERS

```diff
@@ -1 +1 @@
-* @saleor/devtools
\ No newline at end of file
+* @saleor/appstore
```

Comment:
This line has been underlined in my review view - GH suggested to double check if the name is right

Reply:
fixed, it was repo setting